dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -71,6 +76,14 @@ def post_using_text_mode
     render json: {using_text_mode: !!@user.using_text_mode}
   end
 
+  # POST /api/v1/users/<user_id>/using_dark_mode
+  def post_using_dark_mode
```

Comment:
`post` is a little too much information to have in this method name. It relates directly to the HTTP verb that eventually routed here. If we later decide to update this to use the `PATCH` verb, for example, we'd need to update this method name as well. Prefer a more generic name such as `update` or perhaps `set`

Reply:
As a side note, `get` is an overloaded term that can both refer to "get information" similar to a "getter" as well as the HTTP verb "GET", so that method name doesn't need to be updated.

Reply:
Thanks for the info! Updating it now!